app/src/main/java/com/example/touchcar/presentation/model/NetworkSource.kt

```diff
@@ -0,0 +1,18 @@
+package com.example.touchcar.presentation.model
+
+import android.os.Parcelable
+import kotlinx.parcelize.Parcelize
+
+@Parcelize
+data class NetworkSource(
+    val baseUrl: String,
+    val innerUrl: String,
+) : Parcelable {
+
+    val url: String by lazy {
+        StringBuilder().apply {
+            append(baseUrl)
+            append(innerUrl)
+        }.toString()
+    }
```

Comment:
Так как мы тут всего две строки складываем, то оптимизации StringBuilder не даст. Давай упростим конструкцию:
```suggestion
    val url: String = baseUrl + innerUrl
```